http-plugins/pom.xml

```diff
@@ -53,6 +53,12 @@
       <groupId>co.cask.cdap</groupId>
       <artifactId>hydrator-test</artifactId>
     </dependency>
+    <dependency>
```

Comment:
You should define this in the parent pom `dependencyManagement` section, with scope test there. Then in here, simply have:

```
<dependency>
  <groupId>io.netty</groupId>
  <artifactId>netty-all</artifactId>
</dependency>
```